### ISSUE TITLE
What approach is used to train continuous controller in tensorflow-deepq?

### ISSUE THREAD

**Issue by iaroslav-ai** [nivwusquorum/tensorflow-deepq#15]
Did you use some specific paper to write the continuous controller code? I would like to take a look at your continuous branch, and would like to look at such paper first beforehand :) Did you use something like [this?](http://www0.cs.ucl.ac.uk/staff/d.silver/web/Applications_files/ddpg.pdf)

**Comment by siemanko** [comment]
Yes, that's precisely the paper. 

Although now I am focusing on this one - http://arxiv.org/abs/1602.01783 - it is much better.

I did not get it to work yet, hence it's on the branch.

**Comment by iaroslav-ai** [comment]
Looks quite cool. So you want to use A3C for continuous branch? Exponent in output density function and logarithm in policy gradient should cancel nicely ... they use normal density function in policy gradient instead of zero probability for real output, right? There is some space for misinterpreting this in the paper if you ask me :)

**Comment by iaroslav-ai** [comment]
Your current code in continuous branch is for A3C, is this correct?

**Comment by siemanko** [comment]
Yeah, so that's precisely my line of reasoning. I believe they assume normal. 

My current continuous branch attempts to implement continuous dqn.

My new project called deeprl reimplements the entire thing and adds A3C. But it's work in progress..

**Comment by iaroslav-ai** [comment]
Ok thx :)

**Comment by andrewliao11** [comment]
Hi @iaroslav-ai , not sure if you're interested....
I just implement a continuous version of a3c in pytorch [here](https://github.com/andrewliao11/pytorch-a3c-mujoco). 
Hope you'll like it! 😃
